connaisseur/tests/integration/integration-test.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+set -euo pipefail
+
+envsubst <connaisseur/tests/integration/values.yaml >helm/values.yaml
+
+echo "Installing Connaisseur..."
+make install || { echo "Failed to install Connaisseur"; exit 1; }
+echo "Successfully installed Connaisseur"
+
+echo "Testing unsigned image..."
+kubectl run pod --image=connytest/testimage:unsigned >output.log 2>&1 || true
+
+if [[ "$(cat output.log)" != 'Error from server: admission webhook "connaisseur-svc.connaisseur.svc" denied the request: could not find signed digest for image "docker.io/connytest/testimage:unsigned" in trust data.' ]]; then
+  echo "Failed to deny unsigned deployment. Output of deployment:"
```

Comment:
Nitpicky, because it is contained in the output, but I think that the echoed message is not necessarily true - if I am not mistaken the image could also be denied with an unknown error, in which case we'd also want to exit with 1, but we could adapt the echo message to make it clear and prevent contradiction of output and echoed message. If my assumption is correct, I would suggest
```suggestion
  echo "Failed to deny unsigned deployment or denied with an unkown error. Output of deployment:"
```

Reply:
Might also refer to @xopham s comment here https://github.com/sse-secure-systems/connaisseur/pull/42#pullrequestreview-521579294